src/proto/console/text/output.rs

```diff
@@ -118,6 +119,58 @@ impl Output {
     }
 }
 
+impl fmt::Write for Output {
+    fn write_str(&mut self, s: &str) -> fmt::Result {
+        // Allocate a small buffer on the stack.
+        const BUF_SIZE: usize = 128;
+        // Add 1 extra character for the null terminator.
+        let mut buf = [0u16; BUF_SIZE + 1];
+
+        let mut i = 0;
+
+        // This closure writes the local buffer to the output and resets the buffer.
+        let mut flush_buffer = |buf: &mut [u16], i: &mut usize| {
+            buf[*i] = 0;
+            *i = 0;
+
+            self.output_string(buf.as_ptr())
+                .map_err(|_| fmt::Error)
+        };
+
+        {
```

Comment:
I've tested this locally, and we can now remove this `{` because NLL is enabled by default with Rust 2018

Reply:
Done :)